### PR TITLE
xtensa-build-zephyr: use rev2 board TGL-H build of cavs25

### PR DESCRIPTION
**Pull request by kv2019i**
Depends Zephyr commit: https://github.com/zephyrproject-rtos/zephyr/pull/40960

When building for TGL-H (2 DSP cores), we must use revision 2
of the Zephyr cavs25 board definition. Otherwise the core count
is incorrect and resulting firmware image will not boot.

BugLink: https://github.com/thesofproject/sof/issues/5055
Signed-off-by: Kai Vehmanen <kai.vehmanen@linux.intel.com>

### CODE REVIEW

File: scripts/xtensa-build-zephyr.sh
Comment (marc-hb) [inline scripts/xtensa-build-zephyr.sh:200]: Please add a one-line comment with the keywords "board revision" because 1. there is no way to guess unless you're pretty familiar with Zephyr already, 2. using a board revision and Kconfig to override (?) and match the number of cores isn't exactly intuitive.

**Review by marc-hb — CHANGES_REQUESTED** [review]
> Depends Zephyr commit: zephyrproject-rtos/zephyr#40960

This is hard-dependency, meaning the build will fail if this one gets merged first, correct?

So this should be a Github draft until zephyrproject-rtos/zephyr#40960 is merged _and_ our "last known good zephyr commit" catches up.

**Comment by kv2019i** [comment]
Nack from Zephyr maintainers, closing this one.

**Pull request closed without merging**
